2nd-gen/opentelemetry-js/functions/tracing.js

```diff
@@ -0,0 +1,28 @@
+const opentelemetry = require("@opentelemetry/sdk-node");
+const {TraceExporter} = require("@google-cloud/opentelemetry-cloud-trace-exporter");
+const {HttpInstrumentation} = require("@opentelemetry/instrumentation-http");
+const {GrpcInstrumentation} = require("@opentelemetry/instrumentation-grpc");
+const {ExpressInstrumentation} = require('opentelemetry-instrumentation-express');
+const {gcpDetector} = require("@opentelemetry/resource-detector-gcp");
+const {
+    CloudPropagator,
+} = require("@google-cloud/opentelemetry-cloud-trace-propagator");
+
+if (!process.env.FUNCTIONS_EMULATOR) {
```

Comment:
can you please add a comment explaining why this turns off telemetry in the emulators? 

Something like:

"Only enable OpenTelemetry if the function is actually deployed. Emulators don't reflect real-world latency"